src/utils/utils.py

```diff
@@ -33,6 +33,51 @@
 _MOSES_DETOKENIZER = MosesDetokenizer()
 
 
+def check_for_previous_checkpoints(serialization_dir, tasks, phase, load_model):
+    task_directory = ""
+    for task in tasks[::-1]:
+        last_checkpoint = find_last_checkpoint_suffix(serialization_dir, phase, task.name)
+        # if we haven't found a task directory yet
+        if last_checkpoint is not None and len(task_directory) == 0 and phase == "target_train":
+            task_directory = task.name
+    checkpoint_pattern = os.path.join(serialization_dir, task_directory, "*_{}_*.th".format(phase))
```

Comment:
To be safer, check that there's a set of four matching checkpoints that are all from the same epoch. That'll help get around #169.